docs/product/decisions/0008-sfsac-all-fields.md

```diff
@@ -0,0 +1,32 @@
+# 1. Require all fields in SF-SAC XLSX uploads
+
+Introduced: 2023-04-10
+
+## Status
+For consideration
```

Comment:
```suggestion
Approved
```

If this makes it through the PR process and gets merged, then at that point by definition it's approved, and we wouldn't want to have to make another PR just to change this line. ;)